src/diffusers/pipelines/oms/utils.py

```diff
@@ -0,0 +1,72 @@
+import torch.nn.functional as F
```

Comment:
all of this can be moved to utils folder or a version of this might already be there, like is_torch2_available is a 100% in the utils folder somewhere just import things from there in the `oms_diffusion_pipeline` or wherever you need these